Hymns/Audio Player/AudioPlayerView.swift

```diff
@@ -2,11 +2,12 @@ import SwiftUI
 import AVFoundation
 import Combine
 
+// https://github.com/ChrisMash/AVPlayer-SwiftUI
+// https://medium.com/flawless-app-stories/avplayer-swiftui-part-2-player-controls-c28b721e7e27
+// TODO For some reason the combine stuff isn't working with our URLS that aren't straight up mp3 urls for example http://www.hymnal.net/en/hymn/h/894/f=mp3 that is coming from musicJson. However, the combine works when the url is a direct mp3 url such as https://www.hymnal.net/Hymns/NewSongs/mp3/ns0767.mp3
 struct AudioView: View {
 
-    //A second state variable play button toggle is necessary because currentlyplaying is also in control of graying out the music player and working with seeking. We don't want the music player to be greyed out and unavailable everytime we pause the music.
     @State var currentlyPlaying = false
```

Comment:
@finben This seems to be unnecessary and it looks like it's working. Please let me know this is incorrect, according to your understanding.